roles/sap_hypervisor_node_preconfigure/tasks/platform/redhat_ocp_virt/enable-cpumanager.yml

```diff
@@ -0,0 +1,61 @@
+---
+- name: Label nodes
+  ansible.builtin.command: "oc label node {{ __sap_hypervisor_node_preconfigure_register_worker.name }} \
+                            feature.node.kubernetes.io/cpu-feature-invtsc=true --overwrite=true"
+  register: __sap_hypervisor_node_preconfigure_register_label_node
+  changed_when: __sap_hypervisor_node_preconfigure_register_label_node.rc != 0
+
+- name: Enable CPU Manager by patching MCP of "{{ __sap_hypervisor_node_preconfigure_register_worker.name }}"
```

Comment:
AFAIK cpumanager does not need to be enabled manually when installing HCO / CNV Virtualization. It should be enabled automatically and there might be a way to configure `reservedSystemCPUs` later.

Reply:
Well, that's not correct for the OCP versions I've tested (up to 4.14).